### PR TITLE
Use Collapsible component instead of Accordion for CartEntryItem

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
- Switch to Collapsible instead of Accordion component
- Break out to new file

https://user-images.githubusercontent.com/6661511/232848608-99caf019-1bd4-4358-84a3-0df13e29a800.mov

## Justify why they are needed
A Collapsible is better for this use case

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2165** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2165" target="_blank">Graphite</a> 
    * **PR #2180** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2180" target="_blank">Graphite</a> 
      * **PR #2183** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2183" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2183?utm_source=stack-comment).

File: apps/store/src/components/CartInventory/CartEntryItem/CartEntryCollapsible.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import * as Collapsible from '@radix-ui/react-collapsible'
+import { motion } from 'framer-motion'
+import { useTranslation } from 'next-i18next'
+import React, { ReactNode, useState } from 'react'
+import { ChevronIcon, Text, theme } from 'ui'
+
+type Props = { price: string; children: ReactNode }
+
+export const CartEntryCollapsible = ({ price, children }: Props) => {
+  const [open, setOpen] = useState<boolean>(false)
```
Comment (robinandeer) [inline apps/store/src/components/CartInventory/CartEntryItem/CartEntryCollapsible.tsx:11]: Just something to think about -- we will need a way to set the default open value

If you go straight to checkout we want to show this opened by default :)

**Review by robinandeer — APPROVED** [review]
Looks great to me!

**Comment by gustaveen** [comment]
[Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2183) rebased this pull request as part of a merge.

**Comment by gustaveen** [comment]
@gustaveen queued this pull request to merge with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2183).

**Pull request merged**